go.mod

```diff
@@ -1,8 +1,13 @@
 module golang.design/x/clipboard
 
-go 1.16
+go 1.20
```

Comment:
Personally, I'd like to go for a 1.20 upgrade, but the 1.16 version requirement is used to maximize the possible users. If we go up to 1.20, users who are still in 1.19 will not be able to use this package

Reply:
If we check golang.org/x/image, it only requires 1.12 without issues.

Reply:
Ah, ok... I checked golang.org/x/sys and the newer version I referenced requires go 1.17.
The commit [34ab1303b554](https://cs.opensource.google/go/x/mobile/+/34ab1303b554208641230536476d5bb2f750d12e:go.mod) that was used before had go 1.11. But that is also not listed as version on https://pkg.go.dev/golang.org/x/mobile?tab=versions 
Not sure why there are 3 specific commits that were selected as published versions (the previously used one is not among them). 
So what should we do? Have the latest available x/mobile version and stay on go 1.16 for the clipboard lib?
Or go to the latest available x/mobile and update clipboard to 1.17?

Reply:
If we go up to 1.17, then there are some additional work to get rid of this code https://github.com/golang-design/clipboard/blob/main/internal/cgo/handle.go

Reply:
So that code is you contributed to the runtime/cgo standard library starting with go 1.17, and now this local implementation could be removed in favor of using the standard library, did I understand that right?

I could try to do that, but I'm on a Mac, and since this is only used on linux I don't have the right setup to also run the tests for linux...